### DISCUSSION TITLE
各个版本的mips/mipsle的release在老毛子下运行后无任何反应就退出了

### DISCUSSION THREAD

**Discussion by scdx26** [AlistGo/alist#489] [Q&A]
如图所示，从2.0.9到2.0几乎所有的版本我都测试了，运行结果如下，无任何提示，也无日志：

![image](https://user-images.githubusercontent.com/81733370/152641803-80dcaa01-13dd-472f-afd8-87b4b38b0aee.png)

不知道是我的系统问题，还是程序的问题，但我运行 aliyundrive-webdav 没问题。有没有遇到相同问题的朋友

**Comment by xhofe** [comment]
`Padavan`是`uClibc`吧，并未提供prebuilt。

**Reply by scdx26** [reply]
能运行 n2n, tinc, aliyundrive-webdav 这些程序的mipsle版本，我以为alist的mipsle也可以，可能是我理解错了。

**Reply by xhofe** [reply]
release里只提供了`glibc`和`musl-libc`的版本，`uClibc`未提供，可以自行build。

**Reply by scdx26** [reply]
> release里只提供了`glibc`和`musl-libc`的版本，`uClibc`未提供，可以自行build。

好的，谢谢。

**Reply by dalaoshi777** [reply]
openwrt下aarch64 a53能直接运行嘛，我尝试在红米ax6的openwrt上运行alist-linux-arm64，没报错，也没反映

**Reply by scdx26** [reply]
你试试看吧，我感觉可能不行。不知道作者编译好的mips版本有没有应用成功的案例。
